### DISCUSSION TITLE
🔗 Free scroll for the Carousel (Merged with #24814)

### DISCUSSION THREAD

**Discussion by db151** [elementor/elementor#16025] [Feature Request]
With instagram more and more widespread it would be nice to have a carousel offetto with movement without stopping at a fixed point, but free that stops where at any point. The desired effect would be like the one on instagram from mobile (featured stories)
<a href='https://postimages.org/' target='_blank'><img src='https://i.postimg.cc/yYvHWJD4/Cattura-1.jpg' border='0' alt='Cattura-1'/></a>

**Comment by nicholaszein** [comment]
Hello, this request is being merged with #24814 which has more details.
